src/effects/bombette_breaking.c

```diff
@@ -1,17 +1,41 @@
 #include "common.h"
+#include "model.h"
 #include "effects_internal.h"
 
 void bombette_breaking_appendGfx(void* effect);
+void bombette_breaking_init(EffectInstance* effect);
+void bombette_breaking_update(EffectInstance* effect);
+void bombette_breaking_render(EffectInstance* effect);
+
+extern Gfx D_090018C0[];
+extern Gfx D_09001D00[];
+extern Gfx D_09001CC8[];
+extern Gfx D_09001C80[];
+extern Gfx D_090017D0[];
+
+Gfx* D_E0084E10[] = { D_090018C0, D_090018C0, D_090018C0 };
+Gfx* D_E0084E1C[] = { D_09001D00, D_09001CC8, D_09001C80 };
+Gfx* D_E0084E28[] = { D_090017D0, D_090017D0, D_090017D0 };
+
+void (*D_E0084E34)(u16, f32*, f32*, f32*, f32*, f32*, f32*) = get_model_center_and_size;
```

Comment:
neat, more shim functions - except specific to one effect

Reply:
wait actually can't you just call these functions? D_E0084E3C is func_E0084E3C (you can put a comment above the func name saying it's a copy of xyz func)

Reply:
nm I can't read. shim functions lol